spyder/plugins/editor/confpage.py

```diff
@@ -325,3 +325,5 @@ def enable_tabwidth_spin(index):
         vlayout = QVBoxLayout()
         vlayout.addWidget(tabs)
         self.setLayout(vlayout)
+        self.setStyleSheet("QTabWidget::tab-bar {"
```

Comment:
Let's define this style sheet in a centralized location so we can import related things from there.

We could use `spyder/config/gui.py` or create a new file either at

`spyder/config/style.py` or at `spyder/utils/style.py`

Suggestions @ccordoba12 ?

Reply:
For now I think it's fine to apply it on a widget basis.

@goanpeca, once you refactor things, then I agree with moving this to `spyder/utils/style.py`.

Reply:
@juanis2112, please try to set this stylesheet rule in the parent class of our config pages, which is called `PluginConfigPage` and it's placed in `spyder/api/preferences.py`.

That way you would avoid to apply it to every config page separately.

Reply:
I was wrong here, as @juanis2112 let me knew yesterday. However, we can apply the stylesheet rule in `show_preferences` of `MainWindow`.